test/integration/universe/wizards-v1/universe-lifecycle/basic-paths/test-suite.ts

```diff
@@ -1,9 +1,14 @@
-import { SCENARIO_1, scenario1 } from './scenario-1';
 import { TEST_SCENARIO } from '../../../../../utils/common-test-stuff/general-utils';
+import { scenario1, SCENARIO_1 } from './scenario-1';
+import { scenario2, SCENARIO_2 } from './scenario-2';
 
 export const BASIC_PATHS_TEST_SUITE = 'basic-paths';
 export const testSuite = (): void => {
   describe(`${TEST_SCENARIO(SCENARIO_1)} Basic Universe setup`, () => {
     scenario1();
   });
+
+  describe(`${TEST_SCENARIO(SCENARIO_2)} Basic Universe with Warper setup`, () => {
```

Comment:
Would be great to stress: Basic Universe with (${PROTOCOL_SUPPORTED_ASSET_CLASSES.ERC721}) Warper from Preset setup.